### ISSUE TITLE
RgRaw returns fewer matches than the default function

### ISSUE THREAD

**Issue by medwatt** [jesseleite/vim-agriculture#9]
Hello. I am using fzf.vim and did created a new command that allows the possibility of using rg to search a directory other than the current one. I was looking for ways to extend it so I can use options like -t when I stumbled upon your plugin. Thanks for this plugin; it seems to do the job. However, for some reason, I am getting few matches with your plugin than the one one I wrote (which is just a very simple modification I did to the default Rg plugin already part of fzf.vim). Can you probably say why this is so ?

Please see [this image](https://i.imgur.com/rX9XltT.png) to see what I mean.

**Comment by jesseleite** [comment]
Thanks! I'm not sure. My plugin just extends fzf.vim's functions as well (most specifically the `fzf#vim#grep()` function). Might be something to do with [the default rg flags being passed](https://github.com/jesseleite/vim-agriculture/blob/master/autoload/agriculture.vim#L33), which differs from the default flags being passed by your plugin?

**Comment by jesseleite** [comment]
Interesting it looks like fzf.vim's `:Rg` uses `--smart-case` by default, and mine doesn't.

**Comment by medwatt** [comment]
> Interesting it looks like fzf.vim's `:Rg` uses `--smart-case` by default, and mine doesn't.

You're right. I just checked. Including --smart-case with RgRaw returns the same count. Thank's for the plugin ones more.

**Comment by jesseleite** [comment]
I may change this default in my plugin.  Not sure how `--smart-case` works with other case-related flags though.  Will leave this open and look into it.  Thanks!
